### DISCUSSION TITLE
Help required to set-up correctly & clarify documentation

### DISCUSSION THREAD

**Discussion by tugdualenligne** [tomMoulard/fail2ban#48] [Q&A]
Hi there,
Many thanks for that work and to those guys from Epita ;-)
I'm not very good with Traefik, but installed correctly the Plugin following your static set-up (TOML file), however I can't get it to work in my containers.
For example, I added this line to my Filebrowser container (and nothing else in the Traefik container docker-compose as I understand it is not required because I configured with the static files), but this does not seem to work (nothing in the logs):
      - "traefik.http.routers.filebrowser.middlewares=my-fail2ban@file"
I reckon I'm not the only one for whom it is not very clear. Could you please clarify this a bit more in the documentation so that after installing the plugin, we know what to do in each container?
Many thanks in advance

**Comment by tomMoulard** [comment]
First thing, [pilot](https://pilot.traefik.io/plugins/605afbaaa5f67ab9a1b0e51c/fail2-ban) contains also a lot of documentation.

But if it still does not answer all of your needs, can you finalize the installation of the plugin and list (or better, open a PR) all of your friction points ?

**Reply by tomMoulard** [reply]
As a side note, I made it work on my [home setup](https://github.com/tomMoulard/make-my-server/tree/master/traefik). It might help you to set it up correctly
